sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubAuthenticationPolicy_token.cs

```diff
@@ -26,22 +26,23 @@ internal partial class WebPubSubAuthenticationPolicy : HttpPipelineSynchronousPo
         /// <param name="audience"></param>
         /// <param name="claims"></param>
         /// <param name="key"></param>
-        /// <param name="expireAfter"></param>
+        /// <param name="expiresAtUtc">DateTime.Kind must be DateTimeKind.Utc.</param>
         /// <param name="hmacSha512">SHA512 if true, otherwise SHA256</param>
         /// <returns></returns>
         public static string GenerateAccessToken(
             string audience,
             IEnumerable<Claim> claims,
             AzureKeyCredential key,
-            TimeSpan expireAfter = default,
+            DateTime expiresAtUtc = default,
```

Comment:
Why is this not `DateTimeOffset` ?

Reply:
I feel like DT is better for representing UTC times, i.e. is has explicit support for UTC. Possibly I should change it to DTO to comply with guidelines.

Reply:
After a discussions, we decided to switch to DTO. But I will do it after the JWT implementation is changed. The current JWT library does not take in DTO.

Reply:
We talked about this offline and you are going to move to DTO with the next set of changes.